README.md

```diff
@@ -47,13 +47,33 @@ In your page type layout template use `$ElementalArea` to render the elements to
 
 ## Getting more elements
 
-Note that this module only comes by default with the base element and a Content element. If you need more, take
+Note that this module only comes by default with the base element and a "Content" element. If you need more, take
 a look at some other modules:
 
-* [silverstripe/silverstripe-elemental-blocks](https://github.com/silverstripe/silverstripe-elemental-blocks)
-* [dnadesign/silverstripe-elemental-userforms](https://github.com/dnadesign/silverstripe-elemental-userforms)
-* [dnadesign/silverstripe-elemental-list](https://github.com/dnadesign/silverstripe-elemental-list)
-* [dnadesign/silverstripe-elemental-virtual](https://github.com/dnadesign/silverstripe-elemental-virtual)
+* [dnadesign/silverstripe-elemental](https://github.com/dnadesign/silverstripe-elemental): Text content (built-in)
+* [silverstripe/silverstripe-elemental-blocks](https://github.com/silverstripe/silverstripe-elemental-blocks): File and image block
+* [silverstripe/silverstripe-elemental-blocks](https://github.com/silverstripe/silverstripe-elemental-blocks): Banner with call-to-action and content
```

Comment:
Duplicate of previous line

Reply:
Yes, that's done on purpose. One line per block. Split up the module and I don't have to do it ;)

Reply:
Ref: https://github.com/silverstripe/silverstripe-elemental-blocks/issues/38